CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(IOOpenSlide)
-set(IOOpenSlide_LIBRARIES IOOpenSlide)
```

Comment:
This is required if a module is not header-only.

Reply:
True. Abandoning.